game/CMakeLists.txt

```
@@ -108,55 +108,90 @@ foreach(lib ${OUR_LIBS} SDL2 PangoCairo LibRSVG LibXML++ AVFormat SWResample SWS
 	add_definitions(${${lib}_DEFINITIONS})
 endforeach(lib)
 
-option(ENABLE_MIDI "Enable support for MIDI hardware (e-drums with USB/MIDI connection)." ON)
-if(ENABLE_MIDI)
-	find_package(PortMidi)
+# Activating MIDI hardware (e-drums with USB/MIDI connection)
+set(ENABLE_MIDI AUTO CACHE STRING "Use midi hardware (e-drums with USB/MIDI connection)")
+set_property(CACHE ENABLE_MIDI PROPERTY STRINGS AUTO ON OFF)
+if(NOT DEFINED ENABLE_MIDI OR "AUTO" STREQUAL ${ENABLE_MIDI})
```

```suggestion
if("AUTO" STREQUAL "${ENABLE_MIDI}") 
```
as it is now always set; quotes to behave correctly if and empty content is passed to ENABLE_MIDI

---

fixed